test/functional/dbcrash.py

```diff
@@ -37,45 +23,47 @@
        * submit block to node
        * if node crashed on/after submitting:
          - restart until recovery succeeds
-         - check that utxo matches node3 using gettxoutsetinfo
-'''
+         - check that utxo matches node3 using gettxoutsetinfo"""
 
-class ChainstateWriteCrashTest(BitcoinTestFramework):
+import errno
+import http.client
+import random
+import time
+
+from test_framework.mininode import *
+from test_framework.script import *
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import *
 
+class ChainstateWriteCrashTest(BitcoinTestFramework):
     def __init__(self):
         super().__init__()
-        self.num_nodes = 4
```

Comment:
ok, you've convinced me. I'll leave this in for now.

We can remove the default num_nodes in a future PR (and probably remove the default topology and premine while we're at it.